### PR TITLE
:boom: Add finding a date in the future

### PR DESCRIPTION
**Pull request by jakebrinkmann**
@stevemao Hi I added the `date` command for todays date (and a future one)

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -195,13 +196,29 @@ find my_folder -mtime +5
...
+What is todays date?
+
+```
+date +%m/%d/%Y
+```
+
+What about a week from now?
+
+```
+date -d "+7 days"
```
Comment (stevemao) [inline readme.md]: This doesn't work for me :(
```
$ date -d "+7 days"
usage: date [-jnu] [-d dst] [-r seconds] [-t west] [-v[+|-]val[ymwdHMS]] ...
            [-f fmt date | [[[mm]dd]HH]MM[[cc]yy][.ss]] [+format]
```

**Comment by stevemao** [comment]
Awesome work @jakebrinkmann :)

File: readme.md (hunk 1 above)
Comment (jakebrinkmann) [inline readme.md]: hmmm, from [this link](http://stackoverflow.com/questions/1706882/get-the-date-a-day-before-current-time-in-bash/8521717#8521717) it looks like Linux uses GNU Date, but OSX uses a different date function?

Does this work?
```
date -j -v+7d
```

Comment (stevemao) [inline readme.md]: Looks like it does

```
$ date -j -v+7d
Sat Nov 19 14:32:55 AEDT 2016
```

**Comment by stevemao** [comment]
BTW I'm using https://github.com/mathiasbynens/dotfiles, it should preserve most of the built-in commands though.

**Comment by jakebrinkmann** [comment]
@stevemao yea, it seems there is a difference in the `date` utility that ships with either operating system. I don't think your dotfiles are affecting this. 

I've just added two commands, with a comment for each OS

File: readme.md (hunk 1 above)
Comment (jakebrinkmann) [inline readme.md]: Thank you for testing it, good catch 👍

File: readme.md [hunk 2]
```diff
@@ -203,13 +204,30 @@ find my_folder -mtime +5
...
+
+```
+date +%m/%d/%Y
+```
+
+What about a week from now?
+
+```bash
+date -d "+7 days"                            # On linux
+date -j -v+7d                                # On osx
```
Comment (stevemao) [inline readme.md]: nit if you have time. OSX now is called MacOS.

**Comment by jakebrinkmann** [comment]
ok, formatted to match the newly added `tree` function, too

**Comment by cht8687** [comment]
LGTM, thanks! Merged.

**Pull request merged**